.github/workflows/pr.yml

```diff
@@ -2,11 +2,17 @@ name: Validate PR
 
 on:
   pull_request:
-    branches: [ master ]
+    branches: [master]
     types: [assigned, opened, synchronize, labeled]
 
+# Using env variables for step conditions to pass if vars are not found.
 env:
+  ORG: ${{ secrets.IMAGE_ORG2 }}
+  DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
+  DOCKER_USER: ${{ secrets.DOCKER_USER2 }}
+  DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD2 }}
```

Comment:
Why the "2" suffix on these secrets?

Reply:
That is my bad, it was me testing invalid secrets. :)